<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClientWindow</class>
 <widget class="QWidget" name="ClientWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>633</width>
    <height>483</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Робоча станція</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_13" stretch="0,0,2,1">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="styleSheet">
      <string notr="true">QGroupBox {
	font-weight: bold;
}</string>
     </property>
     <property name="title">
      <string>Налаштування</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0,0,0,0,1">
        <item>
         <widget class="QLineEdit" name="ipInput">
          <property name="text">
           <string>127.0.0.1</string>
          </property>
          <property name="maxLength">
           <number>32767</number>
          </property>
          <property name="placeholderText">
           <string>IP серверу</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="portInput">
          <property name="text">
           <string>27015</string>
          </property>
          <property name="maxLength">
           <number>32767</number>
          </property>
          <property name="placeholderText">
           <string>Порт</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="connectButton">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Підключитися</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="disconnectButton">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Відключитися</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="disconnectedLabel">
          <property name="styleSheet">
           <string notr="true">QLabel {
	color: red;
}</string>
          </property>
          <property name="text">
           <string>Не підключено</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="connectedLabel">
          <property name="styleSheet">
           <string notr="true">color: green;</string>
          </property>
          <property name="text">
           <string>Підключено</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>136</width>
            <height>14</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="styleSheet">
      <string notr="true">QGroupBox {
	font-weight: bold;
}</string>
     </property>
     <property name="title">
      <string>Отримання даних</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0,0,0">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="minimumSize">
           <size>
            <width>25</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>з:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateEdit" name="fromDate">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTimeEdit" name="fromTime">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="minimumSize">
           <size>
            <width>25</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>до:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateEdit" name="toDate">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTimeEdit" name="toTime">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QPushButton" name="recieveButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Отримати</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="proccessingLabel">
          <property name="styleSheet">
           <string notr="true">color: orange;</string>
          </property>
          <property name="text">
           <string>Виконується...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="recievedLabel">
          <property name="styleSheet">
           <string notr="true">color: green;</string>
          </property>
          <property name="text">
           <string>Завершено</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_6">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="styleSheet">
      <string notr="true">QGroupBox {
	font-weight: bold;
}</string>
     </property>
     <property name="title">
      <string>Результат</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string>Тип 1 (КОМ 1, 2)</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QTableWidget" name="type1Table">
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="showGrid">
             <bool>true</bool>
            </property>
            <attribute name="horizontalHeaderVisible">
             <bool>true</bool>
            </attribute>
            <attribute name="verticalHeaderVisible">
             <bool>false</bool>
            </attribute>
            <column>
             <property name="text">
              <string>Дата, час</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>V1, м/с</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>V2, м/с</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>T1, °C</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>T2, °C</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>M1, кг</string>
             </property>
            </column>
           </widget>
          </item>
          <item>
           <layout class="QFormLayout" name="formLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Кількість записів:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLabel" name="type1Count">
              <property name="text">
               <string>0</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_2">
         <attribute name="title">
          <string>Тип 2 (КОМ 3, 4)</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QTableWidget" name="type2Table">
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <attribute name="verticalHeaderVisible">
             <bool>false</bool>
            </attribute>
            <column>
             <property name="text">
              <string>Дата, час</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>V1, м/с</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>T1, °C</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>T2, °C</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>M1, кг</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>L, м</string>
             </property>
            </column>
           </widget>
          </item>
          <item>
           <layout class="QFormLayout" name="formLayout_2">
            <item row="0" column="0">
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Кількість записів:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLabel" name="type2Count">
              <property name="text">
               <string>0</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="styleSheet">
      <string notr="true">QGroupBox {
	font-weight: bold;
}</string>
     </property>
     <property name="title">
      <string>Журнали</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_9">
      <item>
       <widget class="QTabWidget" name="tabWidget_3">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tab_5">
         <attribute name="title">
          <string>Системний</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_10">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPlainTextEdit" name="systemLog">
            <property name="whatsThis">
             <string>system</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_6">
         <attribute name="title">
          <string>Прийом</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_11">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPlainTextEdit" name="recieveLog">
            <property name="whatsThis">
             <string>recieve</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_7">
         <attribute name="title">
          <string>Передача</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_12">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPlainTextEdit" name="sendLog">
            <property name="whatsThis">
             <string>send</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
